slack_bot_main.py

```
@@ -0,0 +1,40 @@
+import asyncio
+import logging
+from core.bot import SlackBotHandler
+from core.workflows import PRWorkflows
```

Ruff (0.12.2)

`core.workflows.PRWorkflows` imported but unused

Remove unused import: `core.workflows.PRWorkflows`

(F401)